controllers/snapshot/snapshot_adapter_test.go

```diff
@@ -60,9 +60,9 @@ var _ = Describe("Snapshot Adapter", Ordered, func() {
 		hasSnapshotPR                             *applicationapiv1alpha1.Snapshot
 		hasInvalidSnapshot                        *applicationapiv1alpha1.Snapshot
 		deploymentTargetClass                     *applicationapiv1alpha1.DeploymentTargetClass
-		integrationTestScenario                   *v1beta1.IntegrationTestScenario
-		integrationTestScenarioWithoutEnv         *v1beta1.IntegrationTestScenario
-		integrationTestScenarioForInvalidSnapshot *v1beta1.IntegrationTestScenario
+		integrationTestScenario                   *v1beta2.IntegrationTestScenario
+		integrationTestScenarioWithoutEnv         *v1beta2.IntegrationTestScenario
```

Comment:
Since we won't have `environment` in ITS, then we may update the name and test for ITS with or without environment to make it clean, how do you think?

Reply:
Yes I was thinking of keeping just integrationTestScenario and removing the withoutEnv one.

Reply:
Updated so we keep only integrationTestScenario.